subxt/src/backend/chain_head/follow_stream_driver.rs

```diff
@@ -432,6 +432,10 @@ where
                     self.sub_id = Some(sub_id);
                     continue;
                 }
+                FollowStreamMsg::Event(FollowEvent::Stop) => {
```

Comment:
This was missing to close down subscriptions when the backend was shutdown...

probably my own fault

Reply:
aight, stopped was ignored because of the reconnecting stuff..

hmm, hard to decide that from the stream i.e, whether to close it on `Stop` or not...